### ISSUE TITLE
(other) Players become invisible while talking to NPC's.

### ISSUE THREAD

**Issue by Ephreaym** [opennox/opennox#671]

**Comment by angrykirc** [comment]
Intended, in Vanilla this is one of effects of Script_Freeze()
Though this could be disabled for Coop games

**Comment by dennwc** [comment]
Yes, I think we can keep all the Freeze effects (so mobs stop reacting to the player), but keep drawing it. I think in that case it doesn't matter if it's a coop or not.
